script.js

```diff
@@ -141,32 +141,43 @@ function HideModal() { // eslint-disable-line no-unused-vars
 }
 
 // #region
-const form = document.getElementsByClassName('contact-form')[0];
-const emailField = document.getElementById('email');
-const setError = (message) => {
-  const errorContainer = form.querySelector('.error');
-  errorContainer.innerHTML = message;
-  errorContainer.classList.add('error');
-};
+// Get the form element from the doc
+const form = document.querySelector('form');
+
+// Add an "submit" event listener to the form element
+form.addEventListener('submit', (event) => {
+  // Prevent the default form submission behavior
+  event.preventDefault();
+  // Create an object to store the form data
+  const formData = {};
+
+  // Loop through each input field and text area in the form
+  form.querySelectorAll('input, textarea').forEach((input) => {
+    // Get the name and value of the input field
+    const [name] = input.name;
+    const [value] = input.value;
+    // Add the name and value to the form data object
+    formData[name] = value;
+  });
```

Comment:
- [ ] kindly as requested by the previous reviewer consider creating a single JavaScript object with all the data from the entire form and save it in local storage below is a hint 
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```
this will help you to always manage to meet the project requirement which will also help you in your future job to deliver what's required

Reply:
This is very redundant since the code above makes a JS object based on the specific fields there are in the form, **if I happen to add a new input field in my HTML, I will manually have to add it on JS if I manually create an object**

But I will change it anyways